crates/nu-parser/src/lex.rs

```diff
@@ -247,21 +247,21 @@ pub fn lex_item(
     }
 
     match &input[(span.start - span_offset)..(span.end - span_offset)] {
-        b"out>" => (
+        b"out>" | b"o>" => (
             Token {
                 contents: TokenContents::OutGreaterThan,
                 span,
             },
             None,
         ),
-        b"err>" => (
+        b"err>" | b"e>" => (
             Token {
                 contents: TokenContents::ErrGreaterThan,
                 span,
             },
             None,
         ),
-        b"out+err>" | b"err+out>" => (
+        b"out+err>" | b"err+out>" | b"o+e>" | b"e+o>" => (
```

Comment:
Are there competing desires to parse spaceless math here @jntrnr? If the `+` is impacting math parsing we could use `oe>` and `eo>`.

Reply:
We talked about it, but I think it'd get tricky with `+` and `-` being allowed in filenames and `*` being a valid glob character. I'm guessing we'll keep with requiring spaces.